go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/cloudwego/kitex v0.6.2-0.20230815060351-88ea60530d40
 	github.com/cloudwego/thriftgo v0.3.0
 	github.com/dubbogo/tools v1.0.9
+	github.com/go-zookeeper/zk v1.0.3
```

Comment:
Consider make registry/zookeeper an independent module to avoid directly dependence of this package if a user is not using this module.

Reply:
Ack.